tests/kani/FunctionContract/simple_ensures_pass.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#[kani::ensures(result == x)]
+fn max(x: u32, y: u32) -> u32 {
+    if x > y {
+        x
+    } else {
+        y
+    }
+}
+
+#[kani::proof]
+fn main() {
+    max(7, 6);
+}
```

Comment:
new line

Reply:
You mean after the function?

Reply:
He means at the end of the file. It's a good practice to add an empty new line at the end of each file.

Reply:
btw, you can configure vscode to do that for you: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files